src/mahoji/commands/chop.ts

```diff
@@ -98,13 +99,37 @@ export const chopCommand: OSBMahojiCommand = {
 			name: 'powerchop',
 			description: 'Set this to true to powerchop. Higher xp/hour, No loot (default false, optional).',
 			required: false
+		},
+		{
+			type: ApplicationCommandOptionType.Boolean,
+			name: 'forestry_events',
+			description: 'Set this to true to participate in forestry events. (default false, optional).',
+			required: false
+		},
+		{
+			type: ApplicationCommandOptionType.String,
+			name: 'twitchers_gloves',
+			description: "Change the settings of your Twitcher's gloves. (default egg nests, optional)",
+			required: false,
+			choices: [
+				{ name: 'Egg nest (Default)', value: 'egg' },
+				{ name: 'Ring nest', value: 'ring' },
+				{ name: 'Seed nest', value: 'seed' },
+				{ name: 'Clue nest', value: 'clue' }
```

Comment:
These should be strictly typed, something like:

````
const twitchers = [...] as const;
type TwitcherType = typeof twitchers[number].id;
```

Reply:
fixed, i added it constants.ts since i saw that's where a good majority of them were, let me know if they should be somewhere else.